src/lightning/pytorch/profilers/simple.py

```diff
@@ -79,16 +79,30 @@ def stop(self, action_name: str) -> None:
 
     def _make_report_extended(self) -> Tuple[_TABLE_DATA_EXTENDED, float, float]:
         total_duration = time.monotonic() - self.start_time
-        report = [
-            (a, np.mean(d), len(d), np.sum(d), 100.0 * np.sum(d) / total_duration)
-            for a, d in self.recorded_durations.items()
-        ]
+        report = []
+
+        for a, d in self.recorded_durations.items():
+            d_tensor = torch.tensor(d)
+            mean_d = torch.mean(d_tensor).item()
```

Comment:
It is not necessary to compute the mean separately if we anyway havae to compute the sum below.

Reply:
Yes you are right, I have updated and committed